src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
Please use upper case according to best practices. Like `blach` -> `BLACK` -> 
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html